build.sbt

```diff
@@ -19,10 +17,14 @@ lazy val toolVersionKey = settingKey[String]("The version of the underlying tool
 
 toolVersionKey := {
   val jsonFile = (resourceDirectory in Compile).value / "docs" / "patterns.json"
-  val toolMap = JSON.parseFull(Source.fromFile(jsonFile).getLines().mkString)
-    .getOrElse(throw new Exception("patterns.json is not a valid json"))
-    .asInstanceOf[Map[String, String]]
-  toolMap.getOrElse[String]("version", throw new Exception("Failed to retrieve 'version' from patterns.json"))
+  val source = Source.fromFile(jsonFile)
+  try {
+    val jsonString = source.getLines().mkString
+    val json = ujson.read(jsonString)
```

Comment:
[sjson](https://github.com/eed3si9n/sjson-new) is already a transitive dependency of sbt itself, can we use it instead of another library for such a small use-case?

Reply:
Now I'm using the already provided sjsonnew library.